gengokit/generator/gen.go

```diff
@@ -31,7 +31,7 @@ func GenerateGokit(sd *svcdef.Svcdef, conf gengokit.Config) (map[string]io.Reade
 	codeGenFiles := make(map[string]io.Reader)
 
 	for _, templPath := range templFiles.AssetNames() {
-		actualPath := templatePathToActual(templPath, sd.PkgName)
+		actualPath := templatePathToActual(templPath, strings.ToLower(sd.Service.Name))
```

Comment:
This is a bit gross now, maybe the `ServiceBaseDir` should come in on `conf`. So in  `truss/main.go` conf.ServiceBaseDir` can be set to `strings.ToLower(sd.Service.Name)`, so it seems less magic here.

Was guessing the lowercase path names might cause something odd like this.

Otherwise this deserves a short comment.